### ISSUE TITLE
How to quick-connect the device through tcpip when the ip assigned randomly everytime 

### ISSUE THREAD

**Issue by gemini4545** [Genymobile/scrcpy#4086]
If the IP is changed everytime when connecting to the network, the --list-display will give all the usded IPs, and of course you can't connect the device through scrcpy --tcpip directly, unless using the option --tcpip==ip address. but it is a little bit tedious. 
Is there any fast ways to quick-connect the device when ip changed, or how to forget the long old IPs in the --list-display

**Comment by LazyDevv** [comment]
I assigned a reserved local IP to my preferred device from my router's DHCP settings. That effectively resolves this issue.

**Comment by gemini4545** [comment]
> I assigned a reserved local IP to my preferred device from my router's DHCP settings. That effectively resolves this issue.

Yeah, it's one of the choice, but sometimes it's hard to go into the router's DHCP settings because it's not the personal router, e.g. in the office.
